src/utils.erl

```diff
@@ -0,0 +1,63 @@
+%%%
+%%% Copyright (C) 2015  Christian Ulrich
+%%% Copyright (C) 2016  Vlad Ki
+%%% Copyright (C) 2017  Dima Stolpakov =)
+%%%
+%%% This program is free software; you can redistribute it and/or
+%%% modify it under the terms of the GNU General Public License as
+%%% published by the Free Software Foundation; either version 2 of the
+%%% License, or (at your option) any later version.
+%%%
+%%% This program is distributed in the hope that it will be useful,
+%%% but WITHOUT ANY WARRANTY; without even the implied warranty of
+%%% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+%%% General Public License for more details.
+%%%
+%%% You should have received a copy of the GNU General Public License along
+%%% with this program; if not, write to the Free Software Foundation, Inc.,
+%%% 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+%%%
+
+%% This implements the "legacy" binary API
+
+-module(utils).
```

Comment:
please come up with a descriptive name for the module and make sure you prefix the name with `mod_pushoff_` to avoid namespace clashes (all module names are global everywhere)